tests/devapps/Telemetry/ClientTelemetryHandler.cs

```diff
@@ -0,0 +1,115 @@
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#if TELEMETRY
+extern alias Client;
+
+using System;
+using System.Collections.Generic;
+using Client::Microsoft.Applications.Events;
+
+namespace Microsoft.Identity.Client.DevAppsTelemetry
+{
+    public class ClientTelemetryHandler
+    {
+        private ILogger logger;
+        private readonly string msalEventNameKey;
+        private readonly string ariaTenantId;
+        private readonly Guid sessionId;
+
+        public ClientTelemetryHandler()
+        {
+            // Aria configuration
+            EVTStatus status;
+            LogManager.Start(new LogConfiguration());
+            LogManager.SetNetCost(REAL_TIME_FOR_ALL[0].Rules[0].NetCost);
+            LogManager.LoadTransmitProfiles(REAL_TIME_FOR_ALL);
+            LogManager.SetTransmitProfile(REAL_TIME_FOR_ALL[0].ProfileName);
+            LogManager.SetPowerState(PowerState.Charging);
+
+            ariaTenantId = TelemetryHandlerConstants.AriaTenantId;
+            logger = LogManager.GetLogger(ariaTenantId, out status);
+
+            sessionId = Guid.NewGuid();
+            msalEventNameKey = TelemetryHandlerConstants.MsalEventNameKey;
+        }
+
+        public void OnEvents(List<Dictionary<string, string>> events)
+        {
+            SetEventProperties(events);
+            UploadEventsToAria();
+        }
+
+        private void SetEventProperties(List<Dictionary<string, string>> events)
+        {
+            Guid scenarioId = Guid.NewGuid();
+            Console.WriteLine("{0} event(s) received for scenarioId {1}",
+                events.Count,
+                scenarioId);
+            foreach (var e in events)
+            {
+                Console.WriteLine("Event: {0}", e[msalEventNameKey]);
+                EventProperties eventData = new EventProperties
+                {
+                    Name = e[msalEventNameKey]
+                };
+
+                eventData.SetProperty(TelemetryHandlerConstants.MsalSessionIdKey, sessionId);
+                eventData.SetProperty(TelemetryHandlerConstants.MsalScenarioIdKey, scenarioId);
+                foreach (var entry in e)
+                {
+                    eventData.SetProperty(entry.Key, entry.Value);
+                    Console.WriteLine("  {0}: {1}", entry.Key, entry.Value);
+                }
+                logger.LogEvent(eventData);
+            }
+        }
+
+        private void UploadEventsToAria()
+        {
+            LogManager.UploadNow();
+            LogManagerProvider.DestroyLogManager(ariaTenantId);
+        }
+
+        private List<TransmitPolicy> REAL_TIME_FOR_ALL = new List<TransmitPolicy>
```

Comment:
please change REAL_TIME_FOR_ALL to _realTimeForAll  (naming conventions)
Also, private members should be at the top of the class.